en/advanced/advanced_position_ctrl_tuning.md

```diff
@@ -0,0 +1,101 @@
+<!-- Fixes page load bug in notation, courtesy https://github.com/GitbookIO/plugin-mathjax/issues/34#issuecomment-349453673 -->
+<script>
+gitbook.events.bind("page.change", function() {
+    MathJax.Hub.Queue(["Typeset",MathJax.Hub]);
+}
+</script>
+
+# Advanced Position Control Tuning
```

Comment:
Title to "Advanced Multicopter Position Control Tuning". (because docs will also have FW tuning guides)

The file also needs to be added to the TOC (SUMMARY.md). However if you are moving this to the user guide would not be done in this PR.